dispatcher/pom.xml

```diff
@@ -155,32 +155,32 @@
                             <execution>
                                 <id>enforce-checksum-of-immutable-files</id>
                                 <goals>
-                                    <goal>display-info</goal>
+                                    <goal>enforce</goal>
```

Comment:
@gknobloch As per [https://experienceleague.adobe.com/docs/experience-manager-cloud-service/implementing/content-delivery/disp-overview.html?lang=en#file-structure](https://experienceleague.adobe.com/docs/experience-manager-cloud-service/implementing/content-delivery/disp-overview.html?lang=en#file-structure) only the _default.host_ in _available_vhosts_ directory is the immutable one (should not be neither changed nor deleted), whereas the symlink pointing to it from _enabled_vhosts_ is not immutable, so it can be either modified (pointing to some other file, although that would be weird, or at least copied as file instead of symlink and modified in place, although duplication is the source of all evil...) or even deleted.

Reply:
@gknobloch BTW. Re symlinks mechanism between enabled and available subdirs - wonder why the same pattern is not kept in WKND's dispatcher farms:
- in Archetype - in [conf.dispatcher.d/enabled_farms](https://github.com/adobe/aem-project-archetype/tree/master/src/main/archetype/dispatcher.cloud/src/conf.dispatcher.d/enabled_farms) the _default.farm_  is a symlink to the _default.farm_ in [conf.dispatcher.d/available_farms](https://github.com/adobe/aem-project-archetype/tree/master/src/main/archetype/dispatcher.cloud/src/conf.dispatcher.d/available_farms)
- in WKND - in [conf.dispatcher.d/enabled_farms](https://github.com/adobe/aem-guides-wknd/tree/master/dispatcher/src/conf.dispatcher.d/enabled_farms) the _default.farm_  is a copy of the _default.farm_ in [conf.dispatcher.d/available_farms](https://github.com/adobe/aem-guides-wknd/tree/master/dispatcher/src/conf.dispatcher.d/available_farms)

Was it done on purpose? Were there any issues with symlinks in farms (compared to the ones in vhosts)?

Reply:
@gknobloch We can also include it in this PR, to have the symlinks in enabled *->available * consistent

Reply:
@gknobloch Done in https://github.com/adobe/aem-guides-wknd/pull/247/commits/04499796b00119522ee9e4d29fb732a9d4d3667e